1_fetch.yml

```diff
@@ -140,6 +144,27 @@ targets:
   1_fetch/out/precip_data.nc:
     command: gd_get('1_fetch/out/precip_data.nc.ind')
 
+
+# -- download snow data --
+  # expects "1_fetch/tmp" to exist
+  fetch_snow_tasks:
+    command: create_fetch_snow_tasks(
+      snow_data_yml_name = I('1_fetch_snow_tasks.yml'),
+      snow_data_tmp_dir = "1_fetch/tmp",
```

Comment:
I think you should wrap this in `I()`, so `I("1_fetch/tmp")` since it is not an `.ind` or a target